tests/SpecTests/RetryableReadsSpecTest.php

```diff
@@ -60,7 +58,7 @@ public function testRetryableReads(stdClass $test, ?array $runOn, $data, string
         }
 
         foreach (self::$skippedOperations as $operation => $skipReason) {
-            if (strpos($this->dataDescription(), $operation) === 0) {
+            if (strpos($this->dataDescription(), (string) $operation) === 0) {
```

Comment:
This is necessary because we don't specify an array shape for `$skippedOperations`, correct?

Reply:
Correct. I removed this in favour of a proper `array<string, string>` annotation on the property.